test-scenarios/package.json

```diff
@@ -27,6 +27,8 @@
     "leader-v1": "npm:ember-auto-import@1.7",
     "leader-v2": "npm:ember-auto-import@1.11",
     "old-analyzer": "npm:ember-auto-import@2.1.0",
+    "ember-bootstrap": "^5.0.0",
```

Comment:
I'd like to not have these added to test-scenarios, because it hides the minimal problem behind another dep, but I've yet to create a reproduction without -- stay tuned.